perfdash/README.md

```diff
@@ -22,6 +22,21 @@ Metrics above are available for all kinds of tests divided into load and density
 Application server runs as a deployment on kubernetes cluster. It is hosted on
 *mungegithub* cluster in *k8s-mungegithub* project.
 
+## How to deploy new version of perf-dash.k8s.io
+
+* Increment TAG in the `Makefile` (for example: 2.10 -> 2.11)
+* Modify the perfdash image version in the `deplyoment.yaml` to be the same as the
+  one specified in the `Makefile`
+* Submit a PR, get required approvals and wait until it's merged
+* In GCP console, select `k8s-mungegithub` project
```

Comment:
Can we squash these two lines (this and one below) into: 
* Configure kubectl to connect to the K8s cluster that hosts perf-dash (`mungegithub` cluster in the `k8s-mungegithub` project).